### PR TITLE
Inbox Component With Bell

### PR DESCRIPTION
**Pull request by RoyalSix**
## Description

[Pull Request Description Here]

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`
https://www.loom.com/share/7c5e7f51afd7458fa568e5461410f12c

### CODE REVIEW

File: src/components/Inbox/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import Message from "../Message";
+import {
+  Body, Container, Footer, Title,
+  Header, Close, SubTitle,
+} from "./styled";
+import "../../style.css";
+//@ts-ignore
+import close from "./close.svg";
```
Comment (rileylnapier) [inline src/components/Inbox/index.tsx:12]: these are going to have issues during the build step, we will need a babel svg plugin

File: src/components/Message/styled.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
```
Comment (rileylnapier) [inline src/components/Message/styled.ts:1]: we should support themes on all these guys too

File: src/components/Message/index.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import {
+  Container, ContentContainer, Title, Body, Icon,
+} from "./styled";
+
+function Message({
+  title, body, icon, onClick,
+}) {
+  return (
+    <Container onClick={onClick}>
```
Comment (rileylnapier) [inline src/components/Message/index.tsx]: lets add a "read" or "unread" property to message

File: src/components/Indicator/index.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Container } from "./styled";
+//@ts-ignore
+import Bell from "./Bell";
+
+
+function Indicator({ onClick }) {
+  return (
+    <Container onClick={onClick}>
```
Comment (rileylnapier) [inline src/components/Indicator/index.tsx:9]: if this is an interactive element lets make it a button so its accessible

Comment (rileylnapier) [reply src/components/Indicator/index.tsx:9]: might be worth adding react-a11y to ensure we are accessible

Comment (RoyalSix) [reply src/components/Indicator/index.tsx:9]: Adding to separate task

File: src/components/Inbox/index.tsx (hunk 1 above)
Comment (rileylnapier) [reply src/components/Inbox/index.tsx:12]: how did we solve this one?

**Review by rileylnapier — DISMISSED** [review]

File: src/components/Message/styled.ts (hunk 2 above)
Comment (RoyalSix) [reply src/components/Message/styled.ts:1]: Done

**Review by rileylnapier — APPROVED** [review]

**Pull request merged**